homeassistant/components/aprilaire/strings.json

```diff
@@ -23,6 +23,59 @@
       "thermostat": {
         "name": "Thermostat"
       }
+    },
+    "sensor": {
+      "indoor_humidity_controlling_sensor": {
+        "name": "Indoor humidity controlling sensor"
+      },
+      "outdoor_humidity_controlling_sensor": {
+        "name": "Outdoor humidity controlling sensor"
+      },
+      "indoor_temperature_controlling_sensor": {
+        "name": "Indoor temperature controlling sensor"
+      },
+      "outdoor_temperature_controlling_sensor": {
+        "name": "Outdoor temperature controlling sensor"
+      },
```

Comment:
A "controlling" sensor sounds strange, what does it reflect? The setpoint?

The name indicates that it's configurable, but it isn't, otherwise it'd be a number entity

Reply:
The "controlling" sensor is what the sensor is called on the Aprilaire side. I believe it is named that way because it is what is "controlling" the behavior of the thermostat, as opposed to auxiliary sensors which are only informational. In this integration, only the "controlling" sensors are supported.

Reply:
Would you mind rechecking the naming? Maybe the info "In this integration, only the "controlling" sensors are supported" would be good to add to the docs and then the sensor naming can be more simplistic.

Reply:
@joostlek Did you mean adding that to the home-assistant.io documentation? Or were you referring to adding a clarifying comment to the code?

Reply:
To the integration docs yes